src/tox/config/__init__.py

```diff
@@ -1895,7 +1915,7 @@ def getargvlist(cls, reader, value, replace=True):
             current_command += line
 
             if is_section_substitution(current_command):
-                replaced = reader._replace(current_command, crossonly=True)
+                replaced = reader._replace(current_command, crossonly=True, unquote_path=False)
```

Comment:
`unquote_path=False` is added liberally to aid reviewing, so all invocations of `_replace(..)` can be seen.
IMO they should removed before being merged.